include/core/sdirk_stage_data.h

```diff
@@ -0,0 +1,81 @@
+// SPDX-FileCopyrightText: Copyright (c) 2019-2020, 2025 The Lethe Authors
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception OR LGPL-2.1-or-later
+
+#ifndef SDIRK_STAGE_DATA_H
+#define SDIRK_STAGE_DATA_H
+
+#include <core/parameters.h>
+#include <core/simulation_control.h>
+
+#include <deal.II/base/exceptions.h>
+
+#include <deal.II/lac/full_matrix.h>
+
+#include <string>
+#include <vector>
+
+using namespace dealii;
+
+struct SDIRKTable
+{
+  FullMatrix<double>  A;
+  std::vector<double> b;
+  std::vector<double> c;
+};
```

Comment:
I would add a brief here that explains how an SDIRK table works. What is the meaning of the A, b and c.
(also why is only A capital and b and c are lower case?) Try to explain why this is this way and/or point to an adequate reference